### DISCUSSION TITLE
Is it possible to create multiple mixed conditions for single name?

### DISCUSSION THREAD

**Discussion by slainless** [chakra-ui/panda#2628] [Q&A]
I'm trying to add condition to dark mode and light mode, something like this:
- dark: if `[data-theme=dark]` or `@media(prefers-color-scheme: dark)` + `[data-theme=system]`
- light: if `[data-theme=light]` or `@media(prefers-color-scheme: light)` + `[data-theme=system]`

**Comment by anubra266** [comment]
@slainless Panda aside, there's no way to merge a media query selector and another selector into one selector in `css`. So this is not possible

**Reply by slainless** [reply]
hmm i don't mean in that way.

mixed conditions is already a thing, quoting from docs, using condition like: `['@media (hover: hover) and (pointer: fine)', '&:hover']` will generate a nested selector inside the media selector. To be specific, i want to generate another set of condition for same condition name, something like this?

```css
@media(prefers-color-scheme: dark) {
  [data-theme=system] & {
    ... dark properties
  }
}

[data-theme=dark] & {
  ... same dark properties
}
```

Well, i know with the current implementation and api, this is not possible. But i'm asking anyway just in case.

**Reply by slainless** [reply]
is this possible to do via plugin instead?
